thesis-de/content-de.tex

```diff
@@ -1730,88 +1746,101 @@ \subsection{Definition eines Graphen}
 als auch der Knoten B mit Knoten A als Kante miteinander
 verbunden (vgl.~\cite{kaiser2008c}).
 
-Auf der Weboberfläche werden dem User oder dem Administrator einzelne Graphen
-von z.B. CPU-Auslastung, RAM-Auslastung oder SSD-Schreibzugriffe angezeigt.
-Diese Graphen werden entweder in Linerar- oder Balkenform dargestellt. Lineare
-Graphen haben eine X- und Y-Achse und besitzen zusätzlich einen Anfang und
-einen wachsenden Endpunkt. Er wird mittels
-Metrik~\ref{section:Begriffserklärung}, welche die Weboberfläche aus den
-Datenhaltungssystemen abgreift, erstellt und kann vom Administrator ebenfalls
-nur für eine definierte Zeitspanne angezeigt werden. Balkendiagramme haben nur
-eine X-Achse, wo der gewünschte Analysewert eingetragen ist und werden z.B. für
-ausgefallene SSDs in einem bestimmten Zeitraum verwendet. Diese werden über die
-Metrik aus dem Datenhaltungssystem abgegriffen und dem Administrator über eine
-Übersicht angezeigt. Ein Graph- bzw. Balkendiagramm visualisiert immer eine
-oder mehrere Timeseries~\ref{section:Begriffserklärung}.
+Innerhalb der Datenvisualisierungslösung werden dem User oder dem Administrator
```

Comment:
vielleicht solltest du mal das ganze Dokument durchschauen und User mit Benutzer austauschen. 
Ist ja schließlich in deutsch hier